src/axom/sidre/interface/SidreTypes.h

```diff
@@ -13,9 +13,21 @@
 #ifndef SIDRETYPES_H
```

Comment:
I see this is a .h (C-style) header.  Consider adding a comment like that at SidreDataTypeIds.h lines 14--17.  Use in C would be a reason to write `typedef` instead of `using` statements.

Reply:
Done.